rocAL/rocAL/include/api/rocal_api_info.h

```diff
@@ -33,6 +33,18 @@ extern "C" int ROCAL_API_CALL rocalGetOutputWidth(RocalContext rocal_context);
 /// \return The height of the ROCAL's output image in pixels. It includes all images in the batch.
 extern "C" int ROCAL_API_CALL rocalGetOutputHeight(RocalContext rocal_context);
 
+///
+/// \param rocal_context
+/// \param buf
+/// \return The width of the ROCAL's output image in pixels
+extern "C" void ROCAL_API_CALL rocalGetOutputResizeWidth(RocalContext rocal_context, unsigned int* buf);
```

Comment:
Why do we need this API?

Reply:
Used to get the resize ROI width and height in python